pkgs/top-level/all-packages.nix

```diff
@@ -8744,6 +8744,8 @@ in
 
   fish = callPackage ../shells/fish { };
 
+  fishPlugins = recurseIntoAttrs (callPackage ../misc/fish-plugins { });
+
   fish-foreign-env = callPackage ../shells/fish/fish-foreign-env { };
```

Comment:
Can you move `foreign-env` into fish-plugins? It has received a special treatment so far, because it's needed to source the NixOS envronment.

Reply:
Kind of tangental, I created https://github.com/bouk/babelfish to avoid having fish-foreign-env in my shell, because it's super slow in my experience. Spawning a prompt takes more than a second. Might be worth integrating in the future.

And I agree we should move ffe into the fishPlugins scope

Reply:
Package moved and module updated :+1: